syntax/std_tst.go

```diff
@@ -37,8 +39,48 @@ func stdTest() rel.Attr {
 			createTestCheckFuncAttr("false", func(v rel.Value) bool { return !v.IsTrue() }),
 			createTestCheckFuncAttr("true", func(v rel.Value) bool { return v.IsTrue() }),
 		),
-		rel.NewNativeFunctionAttr("suite", func(value rel.Value) rel.Value {
-			return nil
+		rel.NewNativeExprFunctionAttr("suite", func(expr rel.Expr, local rel.Scope) (rel.Value, error) {
+			switch expr := expr.(type) {
+			case rel.Value:
+				return rel.None, nil
+			case *rel.SetExpr:
+				errors := []error{}
```

Comment:
`var errors []error`
or using `errors.Wrap(err, message)` might make it easier than assigning to a slice then concatenating the errors

Reply:
The clean solution is a well-formatted test output log. This will do for now.